backend/src/rules/create_game.clj

```diff
@@ -11,16 +11,13 @@
 
 (defn new-game
   "Creates a new game object"
-  ([]
-   (new-game {}))
+  ([] (new-game {}))
   ([ini-config]
    {
-    :players (vec (repeat 2 (new-player (if (contains? ini-config :hand)
-                                            (:hand ini-config)
-                                            hands/default-hand))))
-    :rows (vec (repeat 5 {:limit (if (contains? ini-config :limit)
-                                     (:limit ini-config)
-                                     rows/default-limit)
-                          :cards []}))
+    :players (vec (repeat 2 (new-player (:hand ini-config hands/default-hand))))
+    :rows (vec (reduce
+                 #(concat %1 [{:limit %2 :cards []}])
+                 []
+                 (:limits ini-config rows/default-limit)))
```

Comment:
Readability nitpick: `rows/default-limit` reads like you expect and integer and thus the `reduce` makes no sense. `rows/default-limits`(with an s) would make it clear that it's a collection

Reply:
True. If you see previous commits you'll see I changed how limit configuration works. I changed `:limit` for `:limits` but forgot to do the same with the configs. Gonna do.